### PR TITLE
Add satel integra multiple partitions and switchable outputs

### PR DESCRIPTION
**Pull request by c-soft**
**Description:**

Added multiple partitions support as well as switchable outputs. Configuration has been changed (breaking change).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**
https://github.com/home-assistant/home-assistant/pull/21992

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/satel_integra.markdown [hunk 1]
```diff
@@ -59,16 +61,20 @@ port:
...
+  description: List of the partitions to operate on.
+  required: true
+  type: [integer, list]
+  keys:
+    name:
+      description: Name of the partition.
+      required: true
+      type: string    
+    arm_home_mode:
+      description: The mode in which arm the partition when 'arm home' is used. Possible options are `1`,`2` or `3`. For more information on what are the differences between them, please refer to Satel Integra manual.
```
Comment (MartinHjelmare) [inline source/_components/satel_integra.markdown]: The mode in which the partition is armed when...

Comment (MartinHjelmare) [reply source/_components/satel_integra.markdown]: For more information on what the differences are between them...

File: source/_components/satel_integra.markdown [hunk 2]
```diff
@@ -59,16 +61,20 @@ port:
...
-  default: 1
-  type: integer
-arm_home_mode:
-  description: The mode in which arm Satel Integra when 'arm home' is used. Possible options are `1`,`2` or `3`. For more information on what are the differences between them, please refer to Satel Integra manual.
-  required: false
-  default: 1
-  type: integer
+partitions:
+  description: List of the partitions to operate on.
+  required: true
```
Comment (MartinHjelmare) [inline source/_components/satel_integra.markdown]: This is optional according to the source.

Comment (MartinHjelmare) [reply source/_components/satel_integra.markdown]: The default is an empty dict.

Comment (c-soft) [reply source/_components/satel_integra.markdown]: Indeed, it's not necessary, one can decide to use just inputs/outputs of the alarm without mapping its state into HA.

File: source/_components/satel_integra.markdown (hunk 1 above)
Comment (c-soft) [reply source/_components/satel_integra.markdown]: Fixed.

File: source/_components/satel_integra.markdown (hunk 2 above)
Comment (c-soft) [reply source/_components/satel_integra.markdown]: Fixed.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**